todo_app/test_app.py

```diff
@@ -0,0 +1,36 @@
"6006181200fa415dcdc0bd34\",\"id\":\"6006195ae74f187322fda449\",\"starred\":false,\"url\":\"https://trello.com/b/BwoWDFLv/exercise-2\",\"prefs\":{\"permissionLevel\":\"org\",\"hideVotes\":false,\"voting\":\"disabled\",\"comments\":\"members\",\"invitations\":\"members\",\"selfJoin\":true,\"cardCovers\":true,\"isTemplate\":false,\"cardAging\":\"regular\",\"calendarFeedEnabled\":false,\"background\":\"6005634ee81d726404d39df3\",\"backgroundImage\":\"https://trello-backgrounds.s3.amazonaws.com/SharedBackground/original/bd8b4acda66141bfbfdd2bcba104f930/photo-1610960245237-cfab0dea6be4\",\"backgroundImageScaled\":[{\"width\":80,\"height\":100,\"url\":\"https://trello-backgrounds.s3.amazonaws.com/SharedBackground/80x100/3c6a26e218ea3adedc0738033a885d02/photo-1610960245237-cfab0dea6be4.jpg\"},{\"width\":154,\"height\":192,\"url\":\"https://trello-backgrounds.s3.amazonaws.com/SharedBackground/154x192/3c6a26e218ea3adedc0738033a885d02/photo-1610960245237-cfab0dea6be4.jpg\"},{\"width\":384,\"height\":480,\"url\":\"https://trello-backgrounds.s3.amazonaws.com/SharedBackground/384x480/3c6a26e218ea3adedc0738033a885d02/photo-1610960245237-cfab0dea6be4.jpg\"},{\"width\":768,\"height\":960,\"url\":\"https://trello-backgrounds.s3.amazonaws.com/SharedBackground/768x960/3c6a26e218ea3adedc0738033a885d02/photo-1610960245237-cfab0dea6be4.jpg\"},{\"width\":819,\"height\":1024,\"url\":\"https://trello-backgrounds.s3.amazonaws.com/SharedBackground/819x1024/3c6a26e218ea3adedc0738033a885d02/photo-1610960245237-cfab0dea6be4.jpg\"},{\"width\":1024,\"height\":1280,\"url\":\"https://trello-backgrounds.s3.amazonaws.com/SharedBackground/1024x1280/3c6a26e218ea3adedc0738033a885d02/photo-1610960245237-cfab0dea6be4.jpg\"},{\"width\":1536,\"height\":1920,\"url\":\"https://trello-backgrounds.s3.amazonaws.com/SharedBackground/1536x1920/3c6a26e218ea3adedc0738033a885d02/photo-1610960245237-cfab0dea6be4.jpg\"},{\"width\":1280,\"height\":1600,\"url\":\"https://trello-backgrounds.s3.amazonaws.com/SharedBackground/1280x1600/3c6a26e218ea3adedc0738033a885d02/photo-1610960245237-cfab0dea6be4.jpg\"},{\"width\":1638,\"height\":2048,\"url\":\"https://trello-backgrounds.s3.amazonaws.com/SharedBackground/original/bd8b4acda66141bfbfdd2bcba104f930/photo-1610960245237-cfab0dea6be4\"}],\"backgroundTile\":false,\"backgroundBrightness\":\"dark\",\"backgroundBottomColor\":\"#0d1310\",\"backgroundTopColor\":\"#0f140f\",\"canBePublic\":true,\"canBeEnterprise\":true,\"canBeOrg\":true,\"canBePrivate\":true,\"canInvite\":true},\"subscribed\":false,\"labelNames\":{\"green\":\"\",\"yellow\":\"\",\"orange\":\"\",\"red\":\"\",\"purple\":\"\",\"blue\":\"\",\"sky\":\"\",\"lime\":\"\",\"pink\":\"\",\"black\":\"\"},\"dateLastView\":\"2021-02-20T22:29:43.464Z\",\"shortUrl\":\"https://trello.com/b/BwoWDFLv\",\"templateGallery\":null,\"premiumFeatures\":[],\"memberships\":[{\"id\":\"6006195ae74f187322fda44d\",\"idMember\":\"6006181200fa415dcdc0bd34\",\"memberType\":\"admin\",\"unconfirmed\":false,\"deactivated\":false}]}]'"
```

Comment:
In trello.py, `get_boards` is called but the response is never used.

So you could delete that function and forget about trying to handle it here.

But also, this isn't mocking the response correctly - response.json.return_value should be a parsed object rather than a string (you have done it correctly below for the lists endpoint).